docs/src/use/configure/migration-guide.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Configuration Migration Guide
+eleventyNavigation:
+    key: migration guide
+    parent: configure
+    title: Configuration Migration Guide
+    order: 8
+---
+
+This guide provides an overview of how you can migrate your ESLint configuration file from the `.eslintrc` format to the new configuration file format, `eslint.config.js`.
+
+To learn more about the `eslint.config.js` file format, refer to [this blog post](https://eslint.org/blog/2022/08/new-config-system-part-2/).
+
+For reference information on these configuration formats, refer to the following documentation:
+
+* [`.eslintrc` configuration files](configuration-files)
+* [`eslint.config.js` configuration files](configuration-files-new)
+
+## Start Using `eslint.config.js`
+
+As of ESLint v9.0.0, the `eslint.config.js` file format is the default configuration file format. If you are using ESLint v9.0.0 or later, you can start using the `eslint.config.js` file format without any additional configuration.
+
+To use `eslint.config.js` with ESLint v8.x.x, place a `eslint.config.js` file in the root of your project **or** set an `ESLINT_USE_FLAT_CONFIG` environment variable to `true`.
+
+## Key Differences between Configuration Formats
```

Comment:
to reviewer: did i miss anything here?

Reply:
I think the globally-ignoring should be highlighted: https://eslint.org/docs/latest/use/configure/configuration-files-new#globally-ignoring-files-with-ignores

as the old `.eslintignore` will be no longer working.

Reply:
A few more differences that we may want to point out:

* The `root` option no longer exist.
* The `files` option cannot be a single string anymore, it must be an array.
* Files whose name starts with a dot (e.g. `.dotfile.js`) are no longer ignored by default.
* The `parser` and `parserOptions` options are now located under the new setting `languageOptions`.
* `parser` must be an object, not a string.
* The `noInlineConfig` and `reportUnusedDisableDirectives` options are now located under the new setting `linterOptions`.
* Environments associated with an ECMAScript version (e.g. `es6`, `es2020`) can be replaced with `languageOptions.ecmaVersion`, other environments must be replaced with `globals`.
* It is typically not necessary to set `parserOptions.ecmaVersion` if `languageOptions.ecmaVersion` is set.
* It is typically not necessary to set `parserOptions.sourceType` if `languageOptions.sourceType` is set.
* The `sourceType` option now supports the new value "commonjs" (`.eslintrc` supports it too, but it was never documented).

Reply:
added a new 'ignoring files' section and an 'additional changes section w this info